From db533b0926f8ebe304d55df312560fcf4041f6a7 Mon Sep 17 00:00:00 2001
From: Fabio Bertinatto <fbertina@redhat.com>
Date: Mon, 5 Jun 2023 10:44:52 -0300
Subject: [PATCH] UPSTREAM: <carry>: Add OpenShift tooling, images, configs and
 docs

Disable CSI mock tests for SELinux and RecoverVolumeExpansionFailure, which
are alpha features and require additional work to get enabled.
---
 openshift-hack/e2e/annotate/rules.go | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/openshift-hack/e2e/annotate/rules.go b/openshift-hack/e2e/annotate/rules.go
index 95f0e889b74..ce2e8c14913 100644
--- a/openshift-hack/e2e/annotate/rules.go
+++ b/openshift-hack/e2e/annotate/rules.go
@@ -17,6 +17,8 @@ var (
 			`\[Feature:SELinuxMountReadWriteOncePod\]`,
 			`\[Feature:PodSchedulingReadiness\]`,
 			`\[Feature:InPlacePodVerticalScaling\]`,
+			`\[Feature:RecoverVolumeExpansionFailure\]`,
+			`\[Feature:SELinux\]`,
 		},
 		// tests for features that are not implemented in openshift
 		"[Disabled:Unimplemented]": {
-- 
2.40.1

